### DISCUSSION TITLE
[Esercitazione 20 dicembre 2023] Esercizio 2

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#380] [Esercizi - Viaggi/Piazza] [answered]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/51ced077-8a55-4bce-8a29-2d71ec7b4555)

**Comment by Elia-Belli** [comment, marked as answer]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/a7968775-2f7f-47f6-822c-04a4eb3ca69e)

**Comment by daromacarlo** [comment]
![Immagine 2024-01-24 230901](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/41204578-5ddb-44f7-83b7-252acd0f9de7)
![Immagine 2024-01-24 230915](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/2cbeb496-0aad-413d-817b-67e7f2d22f32)
![Immagine 2024-01-24 230923](https://github.com/sapienzastudentsnetwork/algebra2324/assets/135441131/0cefb385-dcaa-4a7c-a6c0-8e3a3d9114e1)
